modular_skyrat/master_files/code/_globalvars/maint_loot_common.dm

```diff
@@ -111,7 +112,8 @@ GLOBAL_LIST_INIT(common_loot, list( //common: basic items
 		/obj/item/storage/firstaid/medical = 25,
 		/obj/item/storage/firstaid/o2 = 25,
 		/obj/item/storage/firstaid/regular = 25,
-		/obj/item/storage/firstaid/toxin = 25
+		/obj/item/storage/firstaid/toxin = 25,
+		/obj/item/lead_pipe = 25
```

Comment:
```suggestion
		/obj/item/lead_pipe = 25,
```

Reply:
the code does not like it if you put a comma on the last item in a list

Reply:
> the code does not like it if you put a comma on the last item in a list

Not true, look at many other files we have that does that. Check [here](https://github.com/Skyrat-SS13/Skyrat-tg/blob/36654a220a1588864d30c05377c07423bcc46352/modular_skyrat/modules/modular_vending/code/autodrobe.dm#L64) for an example.